src/otx/core/exporter/exportable_code/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Methods for exportable code."""
+
+# Copyright (C) 2021-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
#
"""Methods for exportable code."""
```

Reply:
Please follow the current docstring and license format for all files. Most of files are needed to be corrected

Reply:
Ok, thanks

Reply:
Thanks, updated